portfolio/src/main/webapp/script.js

```diff
@@ -32,7 +32,7 @@ function loadComments() {
         for(let comment of commentList) { 
             let commentObject = JSON.parse(comment);
             let commentDate = new Date(commentObject.time);
-            historyElement.appendChild(createListItem(commentObject.comment + ", " + commentDate)); 
+            historyElement.appendChild(createListItem(commentObject.comment + ", " + commentDate + "(" + commentObject.sentiment+ ")")); 
```

Comment:
Nit: space

Also, this is a case where it would probably be best to use [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).